spark/dl/src/main/scala/com/intel/analytics/bigdl/python/api/PythonBigDL.scala

```diff
@@ -65,6 +65,12 @@ case class Sample(features: JList[JTensor],
 case class JTensor(storage: Array[Float], shape: Array[Int],
                    bigdlType: String, indices: Array[Array[Int]] = null)
 
+// `Any` of the value can only be a JTensor or JTable
+case class JTable(value: JList[Any])
+
+// `Any` of the value can only be a JTensor or JTable
+case class JActivity(value: Any)
```

Comment:
Why don't we use JList and JTensor?

Reply:
You mean the value of JActivity should be `JList` and `JTensor` ? in that case the semantic would not be the same as the original `Activity`. Anyway, what the python can see is just nested `List` or `JTensor`